pkgs/by-name/de/dell-bios-fan-control/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dell-bios-fan-control";
+  version = "0-unstable-2022-01-19";
+
+  src = fetchFromGitHub {
+    owner = "TomFreudenberg";
+    repo = "dell-bios-fan-control";
+    rev = "27006106595bccd6c309da4d1499f93d38903f9a";
+    hash = "sha256-3ihzvwL86c9VJDfGpbWpkOwZ7qU0E5U2UuOeCwPMR1s=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp dell-bios-fan-control $out/bin
+  '';
+
+  meta = {
+    description = "A user space utility to set control of fans by bios on some Dell Laptops";
```

The probability of this supporting more types in the future, is low; the probabiltiy of a user hoping that their new shiny laptop is supported and getting disappointed is high. 
 
```suggestion
    description = "A user space utility to set control of fans by bios on Dell 9560 Laptops.";
```

---

It does not support just the 9560, it supports a good chunk of the older lattitude/xps lineup (afaik, maybe other ones too), which is why I left it out, but this is a good point, I'll change it to `some older Dell laptops` instead.